README.md

```diff
@@ -3,3 +3,76 @@ Client tools for using CADC services
 
 Build status:
 <a href="https://travis-ci.org/opencadc/cadctools"><img src="https://travis-ci.org/opencadc/cadctools.svg?branch=master" /></a>
+
+# Developers Guide
+
+Requires Python version: 2.7, 3.3, 3.4, 3.5.
+
+Requires pip.
+
+
+## Installing Packages
+
+Run 
+```
+cd cadcutils && pip install -r ./dev_requirements.txt
+cd cadcdata && pip install -r ./dev_requirements.txt
+pip install cadcutils/
+pip install cadcdata/
+```
+
+## Testing packages
+
+### Testing cadcutils
+
+```
+cd ./cadcutils
+pip install -r ./dev_requirements.txt
+python setup.py test
+```
+
+### Testing cadcdata
+
+```
+cd ./cadcdata
+pip install -r ./dev_requirements.txt
+pip install ../cadcutils/
+python setup.py test
+```
+
+## Usage Example
+
```

Comment:
This is the usage example for the library mode. More examples are in the doc of the DataWsClient class.

To use cadc-data directly, one would do:
'cadc-data get GEMINI 00AUG02_002'

more info:
'cadc-data put --help'
'cadc-data get --help'
'cadc-data info --help'